src/EditorFeatures/VisualBasicTest/Diagnostics/NamingStyles/NamingStylesTests_OptionSets.vb

```diff
@@ -0,0 +1,133 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
+Imports Microsoft.CodeAnalysis.NamingStyles
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Simplification
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics.NamingStyles
```

Comment:
can this not be shared with teh C# side?

Reply:
Apart from the language name, it could be. Where should I put the shared code?

Reply:
fixed in 543e6f746845b52ce4b82d223d2fb166682304b0